Readme.md

```diff
@@ -119,15 +119,15 @@ Functions for building python environments:
 1. **mkPython** - builds a python environment for a given `requirements.txt`.
 1. **mkPythonShell** - builds a python environment suitable for nix-shell.
 1. **mkDockerImage** - builds a layered docker image containing a python environment.
-1. **mkNixpkgs** - returns nixpkgs which is conform to the given requirements.
-1. **mkOverlay** - returns an overlay function to make nixpkgs conform to the given requirements  .
+1. **mkNixpkgs** - returns nixpkgs which conform to the given requirements.
+1. **mkOverlay** - returns an overlay function to make nixpkgs conform to the given requirements.
```

Comment:
Here I understand "nixpkgs" to be plural so "[the packages which] conform". Maybe "nixpkgs conforms" would be more correct though?